### PR TITLE
Upgrade Grafana

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Upgrade Grafana

**Related issue(s)**

Relates to #7918

### CODE REVIEW

File: resources/monitoring/charts/grafana/templates/clusterrole.yaml [hunk 1]
```diff
@@ -3,10 +3,7 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   labels:
-    app: {{ template "grafana.name" . }}
```
Comment (hisarbalik) [inline resources/monitoring/charts/grafana/templates/clusterrole.yaml:6]: where are all this label defined/configured?

Comment (suleymanakbas91) [reply resources/monitoring/charts/grafana/templates/clusterrole.yaml:6]: on helpers file: https://github.com/kyma-project/kyma/pull/7927/files#diff-5c9321b7eb26a1ca1f487014857d878dR56

File: tests/end-to-end/upgrade/pkg/tests/monitoring/metrics.go [hunk 2]
```diff
@@ -128,10 +128,18 @@ func (ut *MetricsUpgradeTest) compareMetrics() error {
 	if err != nil {
 		return err
 	}
+	if previous == nil || len(previous) < 1 {
+		return fmt.Errorf("Previous metric is empty")
```
Comment (k15r) [inline tests/end-to-end/upgrade/pkg/tests/monitoring/metrics.go:132]: error strings should not start with capital letters

File: tests/end-to-end/upgrade/pkg/tests/monitoring/metrics.go [hunk 3]
```diff
@@ -128,10 +128,18 @@ func (ut *MetricsUpgradeTest) compareMetrics() error {
...
+	}
+
 	current, err := ut.collectMetrics(time)
 	if err != nil {
 		return err
 	}
+	if current == nil || len(current) < 1 {
+		return fmt.Errorf("Current metric is empty")
```
Comment (k15r) [inline tests/end-to-end/upgrade/pkg/tests/monitoring/metrics.go:140]: error strings should not start with capital letters

File: tests/end-to-end/upgrade/pkg/tests/monitoring/metrics.go [hunk 4]
```diff
@@ -128,10 +128,18 @@ func (ut *MetricsUpgradeTest) compareMetrics() error {
...
 	}
+	if previous == nil || len(previous) < 1 {
+		return fmt.Errorf("Previous metric is empty")
+	}
+
 	current, err := ut.collectMetrics(time)
 	if err != nil {
 		return err
 	}
+	if current == nil || len(current) < 1 {
```
Comment (k15r) [inline tests/end-to-end/upgrade/pkg/tests/monitoring/metrics.go:139]: assume current == nil => len(current) = 0
so checking the lenght is enough

**Pull request merged**